node/pkg/logscribe/app.go

```diff
@@ -0,0 +1,87 @@
+package logscribe
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/db"
+	"bisonai.com/orakl/node/pkg/logscribe/api"
+	"bisonai.com/orakl/node/pkg/logscribe/utils"
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	logsChannelSize             = 10_000
+	DefaultBulkLogsCopyInterval = 3 * time.Second
+)
+
+func Run(ctx context.Context) error {
+	log.Debug().Msg("Starting logscribe server")
+
+	app, err := utils.Setup("0.1.0")
+	if err != nil {
+		log.Error().Err(err).Msg("Failed to setup logscribe server")
+		return err
+	}
+
+	v1 := app.Group("/api/v1")
+	v1.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString("Logscribe service")
+	})
+
+	logsChannel := make(chan []api.LogInsertModel, logsChannelSize)
```

Comment:
would it be okay to define channel as pointer type?
```
chan *[]api.LogInsertModel
```
since you were sending to channel in dereference type
```
logsChannel <- *logEntries
```
it can be as followed I supposed?
```
logsChannel <- logEntries
```